packages/frontend/src/features/scheduler/hooks/useScheduler.ts

```diff
@@ -85,7 +85,7 @@ const getJobStatus = async (
 
 export const pollJobStatus = async (jobId: string) => {
     return new Promise<void>((resolve, reject) => {
-        getJobStatus(
+        void getJobStatus(
```

Comment:
Also out of scope, but for posterity - `getJobStatus` deconstructs a Promise (via `onError` and `onComplete` arguments), which we're then rebuilding here -- seems like we could probably revise that method to just return an actual Promise instead, so that the caller could then `await` or chain the promise.